plugin/trino-memsql/src/main/java/io/trino/plugin/memsql/MemSqlClient.java

```diff
@@ -204,11 +204,12 @@ protected boolean filterSchema(String schemaName)
                         Optional.empty(),
                         Optional.empty());
                 Optional<ColumnMapping> columnMapping = toColumnMapping(session, connection, typeHandle);
-                log.debug("Mapping data type of '%s' column '%s': %s mapped to %s", schemaTableName, columnName, typeHandle, columnMapping);
+                log.debug("Mapping data type of '%s' column '%s': %s mapped to %s", schemaTableName, remoteColumnName, typeHandle, columnMapping);
                 // skip unsupported column types
                 boolean nullable = (resultSet.getInt("NULLABLE") != columnNoNulls);
                 Optional<String> comment = Optional.ofNullable(emptyToNull(resultSet.getString("REMARKS")));
                 if (columnMapping.isPresent()) {
+                    String columnName = fromRemoteColumnName(session.getIdentity(), connection, getIdentifierMapping(), tableHandle, remoteColumnName);
```

Comment:
This was actually a bug in the memsqlclient implementation `fromRemoteColumnName` was not being called.

Reply:
Could you send a new PR for bug fix? PostgreSQL connector might have the same issue.